pkg/cmd/gist/edit/edit_test.go

```diff
@@ -557,6 +557,18 @@ func Test_editRun(t *testing.T) {
 			},
 			wantErr: "gist ID or URL required when not running interactively",
 		},
+		{
+			name: "edit no-file gist (#10626)",
+			opts: &EditOptions{
+				Selector: "1234",
+			},
+			mockGist: &shared.Gist{
+				ID:    "1234",
+				Files: map[string]*shared.GistFile{},
+				Owner: &shared.GistOwner{Login: "octocat"},
+			},
+			wantErr: "No file found in this gist.",
```

Comment:
This also has to be updated to the new error message.

```suggestion
			wantErr: "no file in the gist",
```